pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -91,10 +92,21 @@ for package in *; do
       exit 1
     fi
 
+    source_independent_hash=""
+    nupkg_file=$(echo "$version/$package.$version.nupkg" | tr '[:upper:]' '[:lower:]')
+    stripped_file=$(mktemp tmp-XXXXXXXX.zip)
+    if zip --quiet "$nupkg_file" --output-file "$stripped_file" --temp-path "$TMPDIR" --delete .signature.p7s ; then
+        source_independent_hash=$(nix-hash --type sha256 --sri "$stripped_file")
+        rm "$stripped_file"
+    else
```

Comment:
It's a bit sad that we've duplicated the logic from make-nuget-deps's definition of `fetchNuget`.